crates/nu-cli/src/repl.rs

```diff
@@ -296,13 +344,13 @@ pub fn evaluate_repl(
         );
 
         start_time = std::time::Instant::now();
-        let buffer_editor = get_editor(engine_state, stack, Span::unknown());
+        let buffer_editor = get_editor(engine_state, &mut stack.child_stack(), Span::unknown());
```

Comment:
some things like this might be misleading, it isn't a stack capture, but because `line_editor` has read references to `stack`, `stack.get_unique_mut` will now fail. Thus we pass in a (cheap) child stack.

Reply:
And given the return of `get_editor` is a bunch of strings, the stack isn't held onto here.